client/question/components/MonospaceDisplayModalDirective.js

```diff
@@ -171,26 +171,34 @@ tie.directive('monospaceDisplayModal', [function() {
         });
 
         /**
-         * Close the modal.
+         * Event handler to hide modal after transition animation ends.
+         */
+        $scope.hideModalAfterAnimation = function(event) {
```

Comment:
Consider using `evt` instead of `event` throughout (which I think we do already).

(`event` is a global variable in some versions of IE, so typically I avoid using it.)